media_softlet/linux/common/os/i915/mos_bufmgr.c

```diff
@@ -2450,7 +2440,25 @@ mos_bufmgr_gem_destroy(struct mos_bufmgr *bufmgr)
 
             mos_gem_bo_free(&bo_gem->bo);
```

Comment:
if there is this function call, need to ensure there are no bo_alloc between init and reallocate

Reply:
Reallocate is being done when vaDriverInit is being called. Internally the sequence is:
1) mos_bufmgr_gem_init()
2) mos_bufmgr_get_devid() and obtain hardware specific information
3) mos_gem_realloc_cache() if it's MTL or ARL
 There's no bo alloc being done within the sequence.